src/utils/cell.ts

```
@@ -17,13 +28,41 @@ function renderCell(
 ) {
   const horizontalShift = index * squareArea;
 
-  const context = new GraphicsContext()
-    .rect(horizontalShift, verticalShift, squareArea, squareArea)
+  const base = new GraphicsContext().rect(
+    horizontalShift,
+    verticalShift,
+    squareArea,
+    squareArea
```

ESLint

[error] Insert `,`

(prettier/prettier)